crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -1159,13 +1192,14 @@ impl TryFrom<&CybersourceRouterData<&types::PaymentsIncrementalAuthorizationRout
                 action_list: None,
                 action_token_types: None,
                 authorization_options: Some(CybersourceAuthorizationOptions {
-                    initiator: CybersourcePaymentInitiator {
+                    initiator: Some(CybersourcePaymentInitiator {
                         initiator_type: None,
                         credential_stored_on_file: None,
                         stored_credential_used: Some(true),
-                    },
+                    }),
                     merchant_intitiated_transaction: Some(MerchantInitiatedTransaction {
-                        reason: "5".to_owned(),
+                        reason: Some("5".to_owned()),
+                        original_authorized_amount: None,
```

Comment:
Incase of incremental auth don't we need to pass the original_authorized_amount?

Reply:
@sai-harsha-vardhan

Reply:
@ArjunKarthik We don't have to send `originalAuthorizedAmount` for incremental auth